### ISSUE TITLE
Uniconsent should not be blocked

### ISSUE THREAD

**Issue by rowboatz** [AdguardTeam/AdGuardSDNSFilter#1625]
### Prerequisites

- [X] This site DOES NOT contains sexually explicit material;
- [X] The problem occurs only when using AdGuard DNS or DNS filtering with AdGuard DNS filter, it is not caused by other ad blockers;
- [X] You're using an up-to-date version of AdGuard DNS filter;
- [X] Browser version is up-to-date;
- [X] If a website or an app is broken, disabling AdGuard DNS filter resolves the issue.

### What DNS server do you use?

Other

### Version

NextDNS

### What DNS filters do you have enabled?

AdGuard DNS filter

### What browser or app do you use?

Firefox

### Which device type do you use?

Desktop

### What type of problem have you encountered?

Website or app doesn't work properly

### Add your comment and screenshots

`uniconsent.com` is blocked even though it belongs to a major consent manager platform. Blocking it prevents consent dialogs from appearing on websites (for example on `https://www.football365.com/` and `https://www.planetf1.com/`). It is neither a tracking domain nor an adserver and is required in order to properly express consent on many websites.

### Privacy

- [X] I agree to follow this condition

**Comment by jellizaveta** [comment]
Yes. It is not a tracker, but it can manage the trackers used, as cookies contain various information - both necessary for the functioning of the site on the device, and that used for analytics purposes - unique identifiers that are used to track the visitor's activity on the website, advertising parameters, etc.

**Comment by hagezi** [comment]
> Yes. It is not a tracker, but it can manage the trackers used, as cookies contain various information - both necessary for the functioning of the site on the device, and that used for analytics purposes - unique identifiers that are used to track the visitor's activity on the website, advertising parameters, etc.

That's true, but blocking via DNS often leads to you allowing everything, see:
https://github.com/hagezi/dns-blocklists/wiki/FAQ#cmps
